### PR TITLE
Fixed: SearchBar.textFieldHintColor not respected on iOS

### LINKED ISSUE #1807: Fixed: SearchBar.textFieldHintColor not respected on iOS
As stated in the title, the textFieldHintColor property of SearchBar does not work when it's inside the ActionBar (tested in ActionBar.titleView). If I move the SearchBar outside the ActionBar tag, it works.

Tested on iOS only. As seen in the screenshot, the text color is gray, but should be white:

image

```
<Page.actionBar>
  <ActionBar title="{{ language.matches_title }}">
    <ActionBar.titleView>

      <AbsoluteLayout row="0" id="searchBar-container">
        <SearchBar top="0" id="searchBar" hint="{{ language.searchMatches }}" textFieldHintColor="#ffffff" textFieldBackgroundColor="#48af7a" width="{{ searchBarWidth }}" />
        <StackLayout top="0" class="border" width="{{ searchBarWidth }}" />
        <StackLayout top="43" class="border" width="{{ searchBarWidth }}" />
      </AbsoluteLayout>

    </ActionBar.titleView>
  </ActionBar>
</Page.actionBar>
```

### PR DESCRIPTION
**Pull request by hamorphis**
Resolves #1807

### CODE REVIEW

File: tns-core-modules/ui/search-bar/search-bar.ios.ts [hunk 1]
```diff
@@ -149,6 +146,16 @@ export class SearchBar extends common.SearchBar {
 
         return this.__textField;
     }
+    
+    get _placeholderLabel(): UILabel {
+        if (!this.__placeholderLabel) {
+            if (this._textField){
+                this.__placeholderLabel = this._textField.valueForKey("placeholderLabel");
```
Comment (hshristov) [inline tns-core-modules/ui/search-bar/search-bar.ios.ts]: Is this private API?
If so we should not check-in it.

**Comment by hamorphis** [comment]
run ci

**Comment by hshristov** [comment]
👍

**Comment by hamorphis** [comment]
run ci

**Pull request merged**